applications/incompressible_navier_stokes/beltrami/application.h

```diff
@@ -247,27 +247,29 @@ class Application : public ApplicationBase<dim, Number>
     PostProcessorData<dim> pp_data;
 
     // write output for visualization of results
-    pp_data.output_data.write_output     = this->output_parameters.write;
+    pp_data.output_data.time_control_data.is_active        = this->output_parameters.write;
+    pp_data.output_data.time_control_data.start_time       = start_time;
+    pp_data.output_data.time_control_data.trigger_interval = (end_time - start_time) / 10.0;
     pp_data.output_data.directory        = this->output_parameters.directory + "vtu/";
     pp_data.output_data.filename         = this->output_parameters.filename;
-    pp_data.output_data.start_time       = start_time;
-    pp_data.output_data.interval_time    = (end_time - start_time) / 10;
     pp_data.output_data.write_divergence = false;
     pp_data.output_data.degree           = this->param.degree_u;
 
     // calculation of velocity error
-    pp_data.error_data_u.analytical_solution_available = true;
+    pp_data.error_data_u.time_control_data.is_active        = true;
+    pp_data.error_data_u.time_control_data.start_time       = start_time;
+    pp_data.error_data_u.time_control_data.trigger_interval = (end_time - start_time) / 10.0;
+    pp_data.error_data_u.analytical_solution_available      = true;
     pp_data.error_data_u.analytical_solution.reset(new AnalyticalSolutionVelocity<dim>(viscosity));
-    pp_data.error_data_u.error_calc_start_time    = start_time;
-    pp_data.error_data_u.error_calc_interval_time = (end_time - start_time) / 10;
-    pp_data.error_data_u.name                     = "velocity";
+    pp_data.error_data_u.name = "velocity";
 
     // ... pressure error
-    pp_data.error_data_p.analytical_solution_available = true;
+    pp_data.error_data_p.time_control_data.is_active        = true;
+    pp_data.error_data_p.time_control_data.start_time       = start_time;
+    pp_data.error_data_p.time_control_data.trigger_interval = (end_time - start_time) / 10.0;
+    pp_data.error_data_p.analytical_solution_available      = true;
```

Comment:
`analytical_solution_available` and `time_control_data.is_active` is a redundancy in my opinion and should be avoided.